File: src/TmrOverlay.App/Overlays/TrackMap/TrackMapOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.TrackMap;

internal static class TrackMapOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "track-map",
        DisplayName: "Track Map",
        DefaultWidth: 360,
        DefaultHeight: 360,
        FadeWhenLiveTelemetryUnavailable: true);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/TrackMap/TrackMapOverlayDefinition.swift
import AppKit

enum TrackMapOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "track-map",
        displayName: "Track Map",
        defaultSize: NSSize(width: 360, height: 360),
        fadeWhenLiveTelemetryUnavailable: true
    )
}
